src/main/java/domain/type/FunctionType.java

```diff
@@ -0,0 +1,27 @@
+package domain.type;
+
+import java.util.Arrays;
+
+public enum FunctionType {
+
+    ONE(1),
+    TWO(2),
+    THREE(3);
```

Comment:
```suggestion
public enum FunctionType {

    ORDER(1),
    PAYMENT(2),
    EXIT(3);
```
같은 ENUM의 뜻을 적어주는건 추천해요!